pennylane_lightning/core/src/simulators/lightning_gpu/algorithms/AdjointJacobianGPU.hpp

```diff
@@ -0,0 +1,446 @@
+// Copyright 2022-2023 Xanadu Quantum Technologies Inc. and contributors.
```

Comment:
We should think about how to replace this file with a generic implementation for LightningQubit, LightningKokkos and LightningGPU. I think that should be doable with a fixed API.

Reply:
Let's do it in a follow-up PR